src/GeositeFramework/plugins/launchpad/style.css

```diff
@@ -26,3 +26,41 @@
     list-style-type: none;
     padding: 0;
 }
+
+.truncate-toggle {
+    padding: 2px !important;
+    font-size: 0.7em !important;
+    width: 35%;
+    float: none !important;
```

Comment:
The basic functionality is unchanged if the `button` class is removed, but all the accumulated styling goes with it, so it seems using `!important` is probably the easiest way to go.